### DISCUSSION TITLE
Teleport community edition kubectl getting access denied

### DISCUSSION THREAD

**Discussion by cliverothery** [gravitational/teleport#10852] [Q&A] [answered]
Hi, 

I have configured a cluster as per the getting started guide.  I can connect to teleport, however I get access denied connecting to the Kubernetes

NAME	       	CHART                 	APP VERSION
teleport-cluster	teleport-cluster-8.3.2	8.3.2

I believe the kubernetes config to the db is ok, 
$ tsh kube ls
Kube Cluster Name                        Selected
---------------------------------------- --------
teleport-cluster.dev-cluster          *

When using admin , or access or member, and executing a kubctl command I get denied or groups 
I have tried with local users and github userss

- Error from server: [00] access denied
- Error from server: this user cannot request kubernetes access, has no assigned groups or users

When i login I have a group

> Profile URL:        https://dev-cluster:443
  Logged in as:       myuser
  Cluster:            dev.cluster
  Roles:              admin, auditor
  Logins:             -teleport-nologin-a4c4562c-62bc-4a04-87c5-477a7609b54d
  Kubernetes:         enabled
  Kubernetes groups:  system:masters

The logs have kubernetes groups , in the login, however I see the kubernetes groups below is <nil> 

teleport-cluster-6dbc6bbc97-mlxfb teleport 2022-03-03T17:37:08Z INFO [AUDIT]     cert.create cert_type:user dev-clustser.... code:TC000I ei:0 event:cert.create identity:map[expires:2022-03-04T05:36:48.007860254Z kubernetes_cluster:teleport-cluster.dev-cluster logins:[myuser] roles:[access] route_to_cluster:teleport-cluster.dev-clustrer teleport_cluster:teleport-cluster.dev-cluster traits:map[github_teams:[GIT TP TEAM] **kubernetes_groups:<nil> kubernetes_users:<nil>** logins:[myuser]] usage:[usage:kube] user:myuser] time:2022-03-03T17:37:08.077Z uid:e27094c3-61ae-4289-a589-7922a5ac4916 events/emitter.go:324

A role for access, members, admin have been created and map to system:masters and other kubernetes groups.

**Comment by scottpgallagher** [comment, marked as answer]
I'd try removing the auditor role from your user and trying again. If that doesn't work can you post the what the role looks like for the admin role?

**Reply by cliverothery** [reply]
I created a access role(as cluster admin), and used access login name not admin name. 

  teams_to_logins:
      logins: ["access","auditor"]

**Reply by PallaviPalisetti** [reply]
create a kubernetes access role and assign to users(member.yaml)
https://goteleport.com/docs/kubernetes-access/helm/guides/digitalocean/#create-a-new-kubernetes-role-named-member
